src/staticanalysis/bot/tests/test_reporter_phabricator.py

```diff
@@ -28,10 +28,10 @@
 You can run this analysis locally with:
  - `./mach clang-format -p path/to/file.cpp` (C/C++)
 
-For your convenience, here is a patch that fixes all the clang-format defects: https://diff.url (use it in your repository with `hg import` or `git apply`)
+For your convenience, [here is a patch]({results}/clang-format-PHID-DIFF-abcdef.diff) that fixes all the clang-format defects (use it in your repository with `hg import` or `git apply`)
```

Comment:
Nit: Please add a full stop character at the end.

```suggestion
For your convenience, [here is a patch]({results}/clang-format-PHID-DIFF-abcdef.diff) that fixes all the clang-format defects (use it in your repository with `hg import` or `git apply`).
```